template/pyproject.toml.jinja

```diff
@@ -1,3 +1,158 @@
+"Pillow==11.3.0",  # pyup: != 11.2.0  # https://github.com/python-pillow/Pillow
+{%- if frontend_pipeline == 'Django Compressor' %}
+{%- if windows and not use_docker %}
+"rcssmin==1.1.2 --install-option='--without-c-extensions'",  # https://github.com/ndparker/rcssmin
+{%- else %}
+"rcssmin==1.1.2",  # https://github.com/ndparker/rcssmin
+{%- endif %}
+{%- endif %}
+"argon2-cffi==25.1.0",  # https://github.com/hynek/argon2_cffi
+{%- if use_whitenoise %}
+"whitenoise==6.10.0",  # https://github.com/evansd/whitenoise
+{%- endif %}
+"redis==6.4.0",  # https://github.com/redis/redis-py
+{%- if use_docker or not windows %}
+"hiredis==3.2.1",  # https://github.com/redis/hiredis-py
+{%- endif %}
+{%- if use_celery %}
+"celery==5.5.3",  # pyup: < 6.0  # https://github.com/celery/celery
+"django-celery-beat==2.8.1",  # https://github.com/celery/django-celery-beat
+{%- if use_docker %}
+"flower==2.0.1",  # https://github.com/mher/flower
+{%- endif %}
+{%- endif %}
+{%- if use_async %}
+"uvicorn[standard]==0.35.0",  # https://github.com/encode/uvicorn
+"uvicorn-worker==0.3.0",  # https://github.com/Kludex/uvicorn-worker
+{%- endif %}
+
+# Django
+"django==5.2.6",  # pyup: < 6.0  # https://www.djangoproject.com/
+"django-environ==0.12.0",  # https://github.com/joke2k/django-environ
+"django-model-utils==5.0.0",  # https://github.com/jazzband/django-model-utils
+"django-allauth[mfa]==65.11.2",  # https://github.com/pennersr/django-allauth
+"django-crispy-forms==2.4",  # https://github.com/django-crispy-forms/django-crispy-forms
+"crispy-bootstrap5==2025.6",  # https://github.com/django-crispy-forms/crispy-bootstrap5
+{%- if frontend_pipeline == 'Django Compressor' %}
+"django-compressor==4.5.1",  # https://github.com/django-compressor/django-compressor
+{%- endif %}
+"django-redis==6.0.0",  # https://github.com/jazzband/django-redis
+{%- if use_drf %}
+# Django REST Framework
+"djangorestframework==3.16.1",  # https://github.com/encode/django-rest-framework
+"django-cors-headers==4.8.0",  # https://github.com/adamchainz/django-cors-headers
+# DRF-spectacular for api documentation
+"drf-spectacular==0.28.0",  # https://github.com/tfranzel/drf-spectacular
+{%- endif %}
+{% if use_django_ninja %}
+# Django Ninja
+"django-ninja==1.4.3",  # https://github.com/vitalik/django-ninja
+{% endif %}
+{%- if frontend_pipeline == 'Webpack' %}
+"django-webpack-loader==3.2.1",  # https://github.com/django-webpack/django-webpack-loader
+{%- endif %}
+]
+
+[dependency-groups]
+
+# Local development dependencies (formerly requirements/local.txt)
+local = [
+"Werkzeug[watchdog]==3.1.3",  # https://github.com/pallets/werkzeug
+"ipdb==0.13.13",  # https://github.com/gotcha/ipdb
+{%- if use_docker %}
+"psycopg[c]==3.2.10",  # https://github.com/psycopg/psycopg
+{%- else %}
+"psycopg[binary]==3.2.10",  # https://github.com/psycopg/psycopg
+{%- endif %}
+{%- if use_async or use_celery %}
+"watchfiles==1.1.0",  # https://github.com/samuelcolvin/watchfiles
+{%- endif %}
+
+# Testing
```

Comment:
Testing doesn't seem to have any db drivers - does it need them?

Reply:
This appears to be an issue with comprehension that the headings inside of the local dependency group being different dependency groups per discussion with @hannylicious. Resolving, as this is a non-issue, though recommending that we format the pyproject.toml file to make this more visually clear that these are part of the same dependency group.